src/lib/ui/components/Option/Option.svelte

```diff
@@ -1,22 +1,23 @@
+<svelte:options immutable />
+
 <script lang="ts">
   import { createEventDispatcher, onDestroy } from "svelte";
   import { fade } from "svelte/transition";
+  import { cx } from "@emotion/css";
   import { generateDatas } from "$lib/logic/utils/objects.js";
   import { useForm } from "$lib/logic/stores/form.js";
 
   import type { Option, Props } from "./Option.proptypes.js";
 
   import * as stylesinternal from "./Option.styles.js";
 
-  export let label: Props["label"];
   export let name: Props["name"];
-  export let id: Props["id"];
-  export let value: Props["value"];
   export let context: Props["context"] = "form";
+  export let disabled: Props["disabled"] = false;
   export let datas: Props["datas"] = {};
-  export let a11y: Props["a11y"] = {};
+  export let options: Props["options"] = [];
 
-  let input: Option;
+  let inputs: Record<string, Option> = {};
```

Comment:
🚫 **[eslint]** <[prefer-const](https://eslint.org/docs/latest/rules/prefer-const)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'inputs' is never reassigned. Use 'const' instead.
```suggestion
  const inputs: Record<string, Option> = {};
```